### PR TITLE
Add new Guardian Light product to product catalog API

### PR DESCRIPTION
**Pull request by tjmw**
## What does this change?

Adds a new product to the product catalog API. The `catalog-code.json` has been updated by exporting the current data from S3.

## How to test

I've deployed to CODE and seen the product catalog regenerate successfully, with the output including the new product.

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: modules/zuora-catalog/test/catalog.test.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import prod from './fixtures/catalog-prod.json';
 
 test('catalogSchema', () => {
 	const codeCatalog: ZuoraCatalog = zuoraCatalogSchema.parse(code);
-	expect(codeCatalog.products.length).toEqual(39);
+	expect(codeCatalog.products.length).toEqual(40);
```
Comment (tjmw) [inline modules/zuora-catalog/test/catalog.test.ts:9]: This is expected (a new product has been added).

File: modules/product-catalog/src/zuoraToProductNameMappings.ts [hunk 2]
```diff
@@ -58,6 +59,7 @@ const zuoraCatalogToProductRatePlanKey: Record<string, string> = {
 	Sunday: 'Sunday',
 	Weekend: 'Weekend',
 	Sixday: 'Sixday',
+	'Guardian Light Monthly': 'Monthly',
```
Comment (tjmw) [inline modules/product-catalog/src/zuoraToProductNameMappings.ts:62]: I'm not 100% sure on the values I've added here. @rupertbates any thoughts?

Comment (rupertbates) [reply modules/product-catalog/src/zuoraToProductNameMappings.ts:62]: I think that's fine

**Review by rupertbates — APPROVED** [review]
👍

**Review by AndreaDiotallevi — APPROVED** [review]

**Pull request merged**